draft-ietf-rtcweb-jsep.xml

```diff
@@ -832,7 +837,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>If the attribute includes a "sar=" (sample aspect ratio)
               value set to something other than "1.0", indicating the
               receiver wants to receive non-square pixels, this cannot be
-              satisfied and the sender MUST NOT transmit the encoding.</t>
+              satisfied and the attribute MUST NOT be used.</t>
 
```

Comment:
I think we need to talk about par as well, I would suggest adding:

If the attribute includes a "par=" (picture aspect ratio) value set to something other than the encoder produces, this cannot be satisfied and the attribute MUST NOT be used.